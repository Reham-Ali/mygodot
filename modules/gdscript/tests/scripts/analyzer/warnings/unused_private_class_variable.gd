# GH-72135

var __a
@warning_ignore("unused_private_class_variable")
var __b
@warning_ignore("unused_private_class_variable") var __c
var __d

func test():
	pass
