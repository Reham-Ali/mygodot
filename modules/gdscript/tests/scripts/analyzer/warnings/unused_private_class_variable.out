GDTEST_OK
>> WARNING
>> Line: 3
>> UNUSED_PRIVATE_CLASS_VARIABLE
>> The class variable "__a" that is private is declared but never used in the class.
>> WARNING
>> Line: 7
>> UNUSED_PRIVATE_CLASS_VARIABLE
>> The class variable "__d" that is private is declared but never used in the class.
